manifests/server/instance/systemd.pp

```diff
@@ -0,0 +1,20 @@
+# @param service_name Overrides the default PostgreSQL service name.
+define postgresql::server::instance::systemd (
+  String[1] $service_name = $name,
+) {
+  # Template uses:
+  # - $facts['os']['name']
+  # - $facts['os']['release']['major']
+  # - $service_name
+  # - $port
+  # - $datadir
+  # - $extra_systemd_config
+  systemd::dropin_file { "${service_name}.conf":
```

Comment:
Consider supporting ensuring the state (like `absent`).

Reply:
done